plugins/inputs/intel_powerstat/dto.go

```diff
@@ -7,8 +7,8 @@ type msrData struct {
 	c3                    uint64
 	c6                    uint64
 	c7                    uint64
-	throttleTemp          uint64
-	temp                  uint64
+	throttleTemp          int64
+	temp                  int64
```

Comment:
I saw #11028 get filed, was that because the reported values were wrong? Or not showing negative values? Or something else? Otherwise, isn't this a breaking change?

Reply:
The main problem was that negative values weren't shown. This is not a very common case in real life, but we wanted to show correct temperature even for this kind of data.